tests/config_tests_device_id.toml

```diff
@@ -1,12 +1,9 @@
 [gateway]
-dbus = true
 http = false
-rvi = true
 socket = true
 
 [network]
 http_server = "127.0.0.1:8888"
-rvi_edge_server = "127.0.0.1:9080"
 socket_commands_path = "/tmp/sota-commands.socket"
 socket_events_path = /tmp/sota-events.socket"
```

Comment:
How has this typo survived so long? There's a missing quote here (unrelated to your PR).

Reply:
Oh, I saw that highlighted path, but could not understand why github highlights it. I saw it in two places.